### PR TITLE
fix code.json

### PR DESCRIPTION
**Pull request by padilla410**
modifications requested as part of DOI-USGS review

### CODE REVIEW

File: code.json
Comment (lekoenig) [inline code.json:55]: @padilla410, would it make sense to point these url's to USGS-R for now (as you do with the `homepageURL` ) and then update all lines when we make the transition? (Feel free to just ignore this comment since I don't have all the context for DGEC stuff 😸)

Comment (jesse-ross) [reply code.json:55]: I am curious about this, too. When I create the migration-assistant PR, I don't have the ability to correctly populate these fields with future URLs, so I've been doing the current URL at the time I create the PR. Is that OK? In either case, I think it would be best to be consistent, and either go with all USGS-R or all DOI-USGS.

**Comment by jesse-ross** [comment]
This looks fine. I'll update my code to create PRs with this information filled in. One question: is the description you set at line 41 different from the one at line 5 for a reason? Would it be OK if I just fill in the description from the GitHub API in both places?

File: code.json
Comment (padilla410) [reply code.json:55]: This is a typo!!! Nice catch yall

Comment (padilla410) [reply code.json:55]: Also, I want to point these to USGS-DOI because I am trying to get this in right now

File: code.json
Comment (padilla410) [inline code.json:53]:
```suggestion
    "homepageURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/releases/tag/v0.1.0",
```

**Comment by padilla410** [comment]
> This looks fine. I'll update my code to create PRs with this information filled in. One question: is the description you set at line 41 different from the one at line 5 for a reason? Would it be OK if I just fill in the description from the GitHub API in both places?

I'm going to update this one too

File: code.json
Comment (padilla410) [inline code.json:41]:
```suggestion
    "description": "An example targets pipeline for pulling data from the Water Quality Portal (WQP)",
```

File: code.json
Comment (lekoenig) [reply code.json:55]: @padilla410 and I decided that we will go ahead and use the USGS-DOI links for now even though they don't actually work yet. We'll merge some planned code changes and then go through with the migration in ~1-2 weeks.

Comment (lekoenig) [reply code.json:55]: I'm under the impression that the url's need to point to the raw files (e.g. no 'blob' in the file path). But the `snow-to-flow` repo in DGEC uses the rendered url's for both the license and the disclaimer, so I'll let you decide whether you want to see if it goes through approval as is. 

```suggestion
    "disclaimerURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/raw/main/DISCLAIMER.md",
```

**Review by lekoenig — CHANGES_REQUESTED** [review]
Thanks for leading the charge, @padilla410! This is looking good, I made a couple suggestions in comments. These comments basically request that you check that unchanged fields have the same values in the Development and Ideation sections. In addition to those, can you make the following edits before merging (sorry, I wasn't able to suggest commits directly for these):

- add "WQP" and "water" to the tags
- update the field `metadataLastUpdated` in the Development section to match today's date

**Comment by jesse-ross** [comment]
This looks fine - I approve it with or without the suggested changes!

**Review by jesse-ross — APPROVED** [review]

**Pull request merged**
